conans/client/conf/__init__.py

```diff
@@ -42,6 +42,7 @@
     Visual Studio:
         runtime: [MD, MT, MTd, MDd]
         version: ["8", "9", "10", "11", "12", "14", "15"]
+        toolset: [None, v90, v100, v110, v120, v120_xp, v140, v140_xp, v140_clang_c2, LLVM-vs2014, v141, v141_clang_c2] 
```

Comment:
consider adding v110_xp, v141_xp and LLVM-vs2014_xp as well

[toolsets](https://i.imgur.com/YgzCeyx.png)

Reply:
Sure, thanks